r2/r2/models/builder.py

```diff
@@ -943,7 +943,7 @@ def update_candidates(self, candidates, sorter, to_add=None):
     "qa": operators.desc('_qa'),
     "hot": operators.desc('_hot'),
     "top": operators.desc('_score'),
-    "random": operators.shuffled('_confidence'),
+    "random": operators.shuffled('_random'),
```

Comment:
Are you able to test this change?

It looks like the rest of these strings ("_date", "_controversy", etc) have corresponding properties like https://github.com/reddit/reddit/blob/master/r2/r2/lib/db/thing.py#L685. When are these properties accessed, and do you need to add one for `_random`?